content/java/concepts/strings/terms/replaceFirst/replaceFirst.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.replaceFirst()'
+Description: 'Replaces the first matching substring in a string with the specified replacement string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Characters'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.replaceFirst()`** method replaces the first matching substring in a string with the specified replacement string.
+
+## Syntax
+
+```pseudo
```

Comment:
Since this is a Java entry, this code block should be Java.

Reply:
The syntax is always in a pseudo code block in every other entry. Are you sure that this one should be in Java?

Reply:
In that case, I guess we can keep it as pseudo.